pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -2,207 +2,643 @@
-    - Otherwise, use whatever domain corresponds to the server where the site is being hosted.
-6. Set `urls.site_api` to whatever value you assigned to `urls.site` with `api` prefixed to it, for example if you set `urls.site` to `web:8000` then set `urls.site_api` to `api.web:8000`.
-7. Setup the environment variables listed in the section below.
-
-### Environment variables
-
-These contain various settings used by the bot.
-To learn how to set environment variables, read [this page](../configure-environment-variables) first.
+If you don't wish to use the provided `config.yml` above, you can copy the `config-default.yml` file, rename it, and change whatever values you need. Note that you don't need to specify all items in `config.yml`, just the ones you want to override such as channel ID's. 
 
-The following is a list of all available environment variables used by the bot:
+See [here](../obtaining-discord-ids) for help with obtaining Discord ID's.
 
-| Variable | Required | Description |
-| -------- | -------- | -------- |
-| `BOT_TOKEN` | Always | Your Discord bot account's token (see [Test server and bot account](#test-server-and-bot-account)). |
-| `BOT_API_KEY` | When running bot without Docker | Used to authenticate with the site's API. When using Docker to run the bot, this is automatically set. By default, the site will always have the API key shown in the example below. |
-| `BOT_SENTRY_DSN` | When connecting the bot to sentry | The DSN of the sentry monitor. |
-| `BOT_TRACE_LOGGERS ` | When you wish to see specific or all trace logs | Comma separated list that specifies which loggers emit trace logs through the listed names. If the ! prefix is used, all of the loggers except the listed ones are set to the trace level. If * is used, the root logger is set to the trace level. |
-| `REDIS_PASSWORD` | When not using FakeRedis | The password to connect to the redis database. *Leave empty if you're not using REDIS.* |
-
----
+These are the main sections in `config-default.yml` that need overriding:
 
-If you are running on the host, while not required, we advise you set `use_fakeredis` to `true` in your `config.yml` file during development to avoid the need of setting up a Redis server.
-It does mean you may lose persistent data on restart but this is non-critical.
-Otherwise, you should set up a Redis instance and fill in the necessary config.
-{: .notification .is-warning }
+* `guild.id`
+* `guild.categories`
+* `guild.channels`
+* `guild.roles`
+* `guild.webhooks`
+* `style.emojis`
 
----
+Additionally:
 
-Example `.env` file:
+* Set `urls.site_schema` and `urls.site_api_schema` to `"http://"`.
+* Set `urls.site_api` to `!JOIN ["api.", *DOMAIN]`.
+* Set `urls.snekbox_eval_api` to `"http://localhost:8060/eval"`.
+* At this stage, set `bot.redis.use_fakeredis` to `true`. If you're looking for instructions for working with Redis, see [Working with Redis](#optional-working-with-redis).
```

Comment:
Can we list these in the order that they appear in the config.yml? Small, but helpful change.

Also cc: @ChrisLovering I remember you told me to do extra stuff here once upon a time during the nightmare of setting up Python. I think it's related to this section?